notification/irc.py

```diff
@@ -173,7 +177,10 @@ def send_msg(channel, msg, server='localhost', port='6667', key=None, topic=None
         irc.send('TOPIC %s :%s\r\n' % (channel, topic))
         sleep(1)
 
-    irc.send('PRIVMSG %s :%s\r\n' % (channel, message))
+    if nick_to:
+        irc.send('PRIVMSG %s :%s\r\n' % (nick_to, message))
+    else:
+        irc.send('PRIVMSG %s :%s\r\n' % (channel, message))
```

Comment:
Here, the code is deciding to ignore channel if nick_to is set.  I'm not sure that's correct behaviour.  If channel is set then the user probably intends for the message to be sent to the channel and to the nick_to user.  So probably you should do:
``` python
+    if nick_to:
+        irc.send('PRIVMSG %s :%s\r\n' % (nick_to, message))
+    if channel:
+        irc.send('PRIVMSG %s :%s\r\n' % (channel, message))
```